app/src/main/res/values/strings.xml

```diff
@@ -412,26 +412,26 @@
 
     <!--Format Strings-->
     <string name="hello_client">Hello, %1$s</string>
-    <string name="double_amount">%1$.2f</string>
-    <string name="double_and_string">%1$.2f %2$s</string>
-    <string name="string_and_string">%1$s %2$s</string>
-    <string name="string_and_double">%1$s %2$.2f</string>
-    <string name="string_and_int">%1$s: %2$d</string>
+    <string name="double_amount" translatable="false">%1$.2f</string>
+    <string name="double_and_string" translatable="false">%1$.2f %2$s</string>
+    <string name="string_and_string" translatable="false">%1$s %2$s</string>
+    <string name="string_and_double" translatable="false">%1$s %2$.2f</string>
+    <string name="string_and_int" translatable="false">%1$s: %2$d</string>
     <string name="invalid_amount">Invalid Amount</string>
     <string name="exit_message">Press back again to exit</string>
 
     <string name="package_name" translatable="false">package</string>
     <string name="permission_denied_camera">Permission Denied to use Camera</string>
-    <string name="one">1</string>
```

Comment:
For some language different numerals are used for example in Hindi, Urdu, Arabic, etc. So this must also be kept as translatable.